packages/ipython/patches/0001-IPython-utils-_process_posix.py-import-pexpect-only-.patch

```diff
@@ -0,0 +1,59 @@
+From 8e55af7ed8620698fa63536ceadd874d51c90ec0 Mon Sep 17 00:00:00 2001
+From: Matthias Koeppe <mkoeppe@math.ucdavis.edu>
+Date: Sun, 28 Jan 2024 22:16:36 -0800
+Subject: [PATCH] IPython/utils/_process_posix.py: import pexpect only when
+ needed
+
+---
+ IPython/utils/_process_posix.py | 7 +++++--
+ 1 file changed, 5 insertions(+), 2 deletions(-)
+
+diff --git a/IPython/utils/_process_posix.py b/IPython/utils/_process_posix.py
+index 59b5c2389..a429ba105 100644
+--- a/IPython/utils/_process_posix.py
++++ b/IPython/utils/_process_posix.py
```

Comment:
I wonder if a better solution would be to somehow avoid importing `_process_posix.py` entirely on Emscripten? Since everything in this file is useless for us...

Reply:
Sure, that certainly makes sense. But I'm not sure which of the two solutions has a better chance to be upstreamed. I've submitted the patch upstream in https://github.com/ipython/ipython/pull/14312